code/game/objects/structures/table_frames.dm

```diff
@@ -99,47 +78,54 @@
 	framestackamount = 2
 	resistance_flags = FLAMMABLE
 
-/obj/structure/table_frame/wood/attackby(obj/item/I, mob/user, params)
-	if(istype(I, /obj/item/stack/sheet/wood))
-		var/obj/item/stack/sheet/wood/W = I
-		if(W.get_amount() < 1)
-			to_chat(user, "<span class='warning'>You need one wood sheet to do this!</span>")
-			return
-		to_chat(user, "<span class='notice'>You start adding [W] to [src]...</span>")
-		if(do_after(user, 20, target = src) && W.use(1))
-			make_new_table(/obj/structure/table/wood)
-		return
-	else if(istype(I, /obj/item/stack/tile/carpet))
-		var/obj/item/stack/tile/carpet/C = I
-		if(C.get_amount() < 1)
-			to_chat(user, "<span class='warning'>You need one carpet sheet to do this!</span>")
-			return
-		to_chat(user, "<span class='notice'>You start adding [C] to [src]...</span>")
-		if(do_after(user, 20, target = src) && C.use(1))
-			make_new_table(/obj/structure/table/wood/poker)
-	else
+/obj/structure/table_frame/wood/attackby(obj/item/I, mob/user, params, already_tried)
+	if(!istype(I, /obj/item/stack/tile/carpet) && !istype(I, /obj/item/stack/sheet/wood))
+		already_tried = TRUE
 		return ..()
 
+	var/obj/item/stack/stack = I
+
+	if(stack.get_amount() < 1) //no need for safeties as we did an istype earlier
+		to_chat(user, "<span class='warning'>You need at least one sheet of [stack] to do this!</span>")
+		return
+
+	to_chat(user, "<span class='notice'>You start adding [stack] to [src]...</span>")
+
+	if(!(do_after(user, 50, target = src) && stack.use(1)))
+		return
+
+	if(istype(stack, /obj/item/stack/tile/carpet))
+		make_new_table(/obj/structure/table/wood/poker)
+		return
+
+	make_new_table(/obj/structure/table/abductor)
```

Comment:
Abductor table? This doesn't look right.

Reply:
How the hell did that end up in this file